### PR TITLE
Simple RnD tweaks

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
Moves Cloning to Medical Machinery and Adds a Medical Analyzer to the Protolathe so Robotics can churn out Mediborgs if they wanted to.

Also Moves Telesci stuff to Teleporter Section.

SORRY FOR LAST PR THINGS HAPPEN OK.

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,64 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"
```
Comment (RemieRichards) [inline tgstation.dme]: >FILE_DIR

**Comment by ExcessiveUseOfCobblestone** [comment]
https://github.com/ExcessiveUseOfCobblestone/-tg-station/commit/1ee8f0ce10535834429b0007b7c5df0e758b913
https://github.com/ExcessiveUseOfCobblestone/-tg-station/commit/98561a60c1d1354576730eef47e7837f5883dd01e

Are the ones you want to look at, the rest is basically me messing around and reverting them afterwards :smile:

**Comment by KorPhaeron** [comment]
You fucked up the DME

**Comment by ExcessiveUseOfCobblestone** [comment]
> TFW you can't even copypaste right

OK give me a bit

**Comment by SconesC** [comment]
17 commits for those two changes

My god

**Comment by ExcessiveUseOfCobblestone** [comment]
I like commits ok

**Comment by SconesC** [comment]
Move the telesci junk to bluespace please

**Comment by ExcessiveUseOfCobblestone** [comment]
I just learned about the neat 'branch' feature on github so I'll keep it classy with the commits :wink:

**Comment by Lumipharon** [comment]
>making the circuit printer categories actually make sense

What is this witchcraft?

**Comment by RandomMarine** [comment]
Medical analyzers should be put in the autolathe list instead. They're a common item.

**Comment by ExcessiveUseOfCobblestone** [comment]
Just my personal opinion but I think they fit very closely to the Spectrometers produced in the proto hence why I'd rather them be kept in the proto over the auto.

Plus Proto means Robotists can churn out a few medibots roundstart. Putting it in the auto means they'd have to initially go to Cargo to get medhuds.

**Comment by RandomMarine** [comment]
Spectrometers come only from research (Or lucky artifact rooms) whereas health analyzers are existing tech that's stocked in every first-aid kit. I think they're more of a parallel to analyzers, which come in every blue toolbox and the autolathe can produce.

**Comment by Razharas** [comment]
I agree with random marine
And its not like R&D doesnt have autolathe each game anyway

**Comment by Iamgoofball** [comment]
>making mass spectros
>ever

>not just using the alt mode for health analyzers

**Comment by ExcessiveUseOfCobblestone** [comment]
goof give spectrometers a reason to live pls :)

**Comment by SconesC** [comment]
22 commits later

**Comment by ExcessiveUseOfCobblestone** [comment]
:(

**Comment by Iamgoofball** [comment]
@ExcessiveUseOfCobblestone Why don't we just remove spectrometers? They're shit to use, clunky, and fucking worthless.

**Pull request merged**